windows-installer/template.nsi

```diff
@@ -22,8 +17,12 @@ RequestExecutionLevel admin
  
 SetCompressor lzma
 
-!define MULTIUSER_EXECUTIONLEVEL Highest
-!define MULTIUSER_INSTALLMODE_DEFAULT_CURRENTUSER
+; CERTBOT CUSTOM BEGIN
+; Administrator privileges are required to insert a new task in Windows Scheduler.
+!define MULTIUSER_EXECUTIONLEVEL Admin
+;!define MULTIUSER_EXECUTIONLEVEL Highest
+;!define MULTIUSER_INSTALLMODE_DEFAULT_CURRENTUSER
```

Comment:
Can you add a comment documenting that commenting out this line makes the installer default to installing Certbot for all users? Bonus points for linking to https://nsis.sourceforge.io/Docs/MultiUser/Readme.html or another source.

Reply:
Done.